actioncable/lib/action_cable/channel/test_case.rb

```diff
@@ -0,0 +1,207 @@
+require "active_support/test_case"
+require "active_support/core_ext/hash/indifferent_access"
```

Comment:
We also need to require the top level here with this at the top: `require "active_support"`.